src/components/episodesComponents.tsx

```diff
@@ -0,0 +1,52 @@
+import { formatEpisodeCode } from "./formatEpisodeCode";
+import { cleanUpSummary } from "./cleanUpSummary";
+
+export interface IEpisode {
```

Comment:
Would be best to omit lines in the interface that are not read in the body of the file.